deploy/ovms/values.yaml

```diff
@@ -0,0 +1,12 @@
+replicas: 1
+
+config_configmap_name: ""
+
+model_name: ""
+model_path: ""
+
+gcp_creds_secret_name: ""
+
+aws_access_key_id: ""
```

Comment:
do we want to support also minio? If so, it would be good to added endpoint_url param.

Reply:
added.